classifier_TEMPLATE.yaml

```diff
@@ -113,6 +119,9 @@ tracking:
         cropped_regions_strategy: "cautious"
 
     filters:
+        # regions with a movement less than this are have tracking scored based of x,y matching, rathen than mid points
```

Comment:
how about: "regions with a movement less than this have their tracking scored based of x,y matching, instead of than mid points"